src/main/java/net/dv8tion/jda/webhook/SendWebhookMessage.java

```diff
@@ -0,0 +1,429 @@
+package net.dv8tion.jda.webhook;
+
+import net.dv8tion.jda.client.entities.Group;
+import net.dv8tion.jda.core.JDA;
+import net.dv8tion.jda.core.entities.*;
+import net.dv8tion.jda.core.requests.RestAction;
+import net.dv8tion.jda.core.requests.restaction.AuditableRestAction;
+import net.dv8tion.jda.core.requests.restaction.MessageAction;
+
+import java.time.OffsetDateTime;
+import java.util.Collections;
+import java.util.Formatter;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Represents a message that has been send from a webhook
+ */
+public class SendWebhookMessage implements Message
```

Comment:
Why are you not extending the abstraction we introduced for the other 3 message types?

Reply:
It would make more sense to call this `SentWebhookMessage` instead